app/assets/javascripts/school_administrator_dashboard/dashboard_components/DashboardHelpers.js

```diff
@@ -44,7 +44,7 @@ export default {
 
   schoolYearStart: function() {
     const today = moment();
-    return today.month() < 8 ? today.subtract(1, 'year').year() + "-08-10" : today.year() + "-08-10";
+    return today.month() < 8 ? today.subtract(1, 'year').year() + "-08-10" : today.year() + "-08-15";
```

Comment:
Should these both be the same, whether it's 08-10 or 08-15?

Reply:
@kevinrobinson they certainly should.